packages/api/tests/endpoints/collections/test-put.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
```

Comment:
This should be merged with `update-collection`, which are tests for the same thing